CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -207,24 +232,133 @@ def get_invalid_format_tags(base_column_value, output_column_value):
     invalid_format_tags = []
     if not base_column_value or not output_column_value:
         return invalid_format_tags
-    for tag in FORMATTING_TAGS:
+    for tag in INLINE_FORMATTING_TAGS:
         base_column_matches_count = len(re.findall(tag, base_column_value))
         output_column_matches_count = len(re.findall(tag, output_column_value))
         if base_column_matches_count != output_column_matches_count:
             invalid_format_tags.append(tag)
+    for tag in BLOCK_FORMATTING_TAGS:
+        base_column_matches_count = regex_match_count(tag, base_column_value)
+        output_column_matches_count = regex_match_count(tag, output_column_value)
+        if base_column_matches_count != output_column_matches_count:
+            invalid_format_tags.append(tag)
+
     return invalid_format_tags
 
 
-def checkRowForMismatch(row, columnDict, baseColumnIdx=None, ignoreOrder=False, wsOut=None, mismatchFlagIdx=None,
-                        outputMismatchTypesFlag=False, skipFormatCheckFlag=False, formatCheckCharacters=None,
-                        formatCheckCharactersAdd=None, verbose=False):
+def fix_block_tags_mismatch(baseText, outputText):
+    """
+        Fixes mismatches for headings, ordered list and unordered list
+
+        Input:
+        baseText(str): base column text
+        outputText(str): Comparing column text
+
+        :return: Fixed output text
+        """
+    baseTextLines = baseText.splitlines()
+    outputTextLines = outputText.splitlines()
+    fixed_output_text = []
+
+    # If line count in base text and output text doesn't match,
+    # return outputText as is
+    if len(baseTextLines) != len(outputTextLines):
```

Comment:
what's the reasoning behind this? We ignore it completely and not let user know anything?

Reply:
@mkangia 
Basically if the line count doesn't match we're not attempting any fixes to be on safe side.. This cell will still be marked in Yellow to warn the user.

Reply:
cool, can we update that comment in the code so that it is clear that we are not skipping checks?

Reply:
done